Assets/HoloToolkit/Utilities/Scripts/ManualCameraControl.cs

```diff
@@ -89,7 +109,7 @@ private Vector3 GetCameraControlTranslation()
         deltaPosition += InputCurve(Input.GetAxis("Horizontal")) * this.transform.right;
         deltaPosition += InputCurve(Input.GetAxis("Vertical")) * this.transform.forward;
```

Comment:
I agree. I'll fix/clean all that no worries.
I'll refactor the code as well as we either use the down/up keys or the horizontal/vertical axis so we should multiply by the transform vectors just once